<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="gridFrame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QFrame" name="horizontalFrame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLineEdit" name="searchLine"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="showStandings">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Показать турнирную таблицу</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="aboutuser">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="teamsButton">
             <property name="text">
              <string>Список команд</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="forwardsRanking">
             <property name="text">
              <string>Рейтинг нападающих</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="goalkeeperRanking">
           <property name="text">
            <string>Рейтинг вратарей</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="defenderRankingButton">
           <property name="text">
            <string>Рейтинг защитников</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="allPlayersButton">
           <property name="text">
            <string>Все игроки</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTableWidget" name="tableWidgetPlayers"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="addPlayer">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Добавить игрока</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deletePlayer">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Исключить игрока</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="newMatch">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Новый матч</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addTeam">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Добавить команду</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addUserButton">
             <property name="text">
              <string>Добавить пользователя</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Выйти из аккаунта</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
